traffic_ops/testing/api/v4/profiles_test.go

```diff
@@ -256,7 +270,23 @@ func UpdateTestProfiles(t *testing.T) {
 	if respProfile.Description != expectedProfileDesc {
 		t.Errorf("results do not match actual: %s, expected: %s", respProfile.Description, expectedProfileDesc)
 	}
-
+	if respProfile.Type != expectedType {
+		t.Errorf("results do not match actual: %s, expected: %s", respProfile.Type, expectedType)
+	}
+	if respProfile.CDNID != *expectedCDNId {
+		t.Errorf("results do not match actual: %d, expected: %d", respProfile.CDNID, expectedCDNId)
```

Comment:
This will print the pointer `expectedCDNId`, not the value to which it points. But if you just don't use a pointer, since you don't need to, this problem will go away.

Reply:
Fixed it.